### PR TITLE
feat/more outputs

### PR DESCRIPTION
**Pull request by stevejpurves**
* fixes safe plain text jupyter outputs
* let iframe render errora
* some local dev setup (now need local server for local dev)
* changes output styling

### CODE REVIEW

File: app/myst-to-react/output/jupyter.tsx [hunk 1]
```diff
@@ -70,15 +47,22 @@ export const NativeJupyterOutputs = ({
     return <div className="text-red-500">Error rendering output: {error.message}</div>;
   }
 
+  if (process.env.NODE_ENV === 'development')
+    console.debug('Output connecting to http://localhost:3003');
+  console.log({ height });
   return (
     <>
-      {loading && <div>Loading...</div>}
+      {loading && <div className="p-2.5">Loading...</div>}
```
Comment (rowanc1) [inline app/myst-to-react/output/jupyter.tsx:55]: Had no idea p-2.5 was a thing.?

File: package.json [hunk 2]
```diff
@@ -40,7 +40,8 @@
     "redux": "^4.1.2",
     "redux-thunk": "^2.4.1",
     "remix": "^1.3.3",
-    "swr": "^1.2.2"
+    "swr": "^1.2.2",
+    "thebe-core": "file:.yalc/thebe-core"
```
Comment (rowanc1) [inline package.json:44]: This is yalc-ed in still.

**Review by rowanc1 — COMMENTED** [review]
Just took a scroll through.

File: app/myst-to-react/output/jupyter.tsx (hunk 1 above)
Comment (stevejpurves) [reply app/myst-to-react/output/jupyter.tsx:55]: anything fractional is a rem

**Pull request merged**